tests/test_comm.rs

```diff
@@ -1,105 +1,82 @@
 use common::util::*;
-use std::ffi::OsStr;
 
 static UTIL_NAME: &'static str = "comm";
-fn at_and_ucmd() -> (AtPath, UCommand) {
-    let ts = TestScenario::new(UTIL_NAME);
-    let ucmd = ts.ucmd();
-    (ts.fixtures, ucmd)
-}
 fn new_ucmd() -> UCommand {
     TestScenario::new(UTIL_NAME).ucmd()
 }
```

Comment:
Good idea. We could even combine it all into a single macro, something like test_module!("comm") which would define with ```#[allow(deadcode)]``` at_and_ucmd as well as new_ucmd. I will do another PR for that later.